src/foam/u2/view/TabChoiceView.js

```diff
@@ -34,8 +34,8 @@ foam.CLASS({
 
     ^ label {
       cursor: pointer;
-      line-height: 48px;
-      padding: 0 32px;
+      line-height: 1.4px;
```

Comment:
This seems really small. Readable text isn't less than 1.4px tall, so are you depending on some side effect of a tiny line-height? How should I test this?

Reply:
oops. thanks for the catch. there wasn't supposed to be a px.